vars/scale_clusterdefinition.json

```diff
@@ -72,9 +72,11 @@
       "callhome_schedule": [daily,weekly]
   },
   "scale_protocols":{
-      "smb": false,
-      "nfs": false,
-      "export_ip_pool": [192.168.100.100,192.168.100.101],
+      "smb": true,
+      "nfs": true,
+      "use_ipv6": true,
+      "interface": [eth0,eth1],
```

Comment:
We should give example and `use_ipv6` , please change this flag to `scale_ces_ipv6_enable` . and it should be false